job.ci.build.yml

```diff
@@ -34,10 +34,10 @@ steps:
     jdkVersionOption: 1.11
     mavenPomFile: $(MavenPOMFile)
     testResultsFiles: '**/TEST-*.xml'
-    sonarQubeRunAnalysis: true
+    sonarQubeRunAnalysis: false
```

Comment:
Any reason you've disabled the sonar analysis?

Reply:
Incompatibility with java version.
The report wasn't used anyways.
Added a tech debt ticket to use sonar.

https://uipath.atlassian.net/browse/AUOPS-3476?atlOrigin=eyJpIjoiMDZkNGQ0ZjM3Y2EyNGRjYjk4Zjc3NGZiOTBmYjU4MDkiLCJwIjoiamlyYS1zbGFjay1pbnQifQ

![image](https://github.com/jenkinsci/uipath-automation-package-plugin/assets/108731924/7db6ff34-acfc-41cb-8cce-a540e982180c)